dev-libs/libvformat/libvformat-1.13-r3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Library to read and write vcard files"
+HOMEPAGE="https://sourceforge.net/projects/vformat/"
+SRC_URI="
+	mirror://debian/pool/main/libv/${PN}/${PN}_${PV}.orig.tar.gz
+	mirror://debian/pool/main/libv/${PN}/${PN}_${PV}-12.2.debian.tar.xz
```

Comment:
I can't fetch this.
`!!! Couldn't download 'libvformat_1.13-12.2.debian.tar.xz'. Aborting.`